servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ToStH1Utils.java

```diff
@@ -249,7 +250,7 @@ static Http2Headers h1HeadersToH2Headers(HttpHeaders h1Headers) {
         for (Map.Entry<CharSequence, CharSequence> h1Entry : h1Headers) {
             // header field names MUST be converted to lowercase prior to their encoding in HTTP/2
             // https://tools.ietf.org/html/rfc7540#section-8.1.2
-            http2Headers.add(h1Entry.getKey().toString().toLowerCase(), h1Entry.getValue());
+            http2Headers.add(h1Entry.getKey().toString().toLowerCase(Locale.ENGLISH), h1Entry.getValue());
```

Comment:
I wonder how this changes the behavior compared to the original method which picked the default locale?

Reply:
Should not impact anything because header names are defined by RFC as:
```
header-field = field-name ":" OWS field-value OWS
field-name = token
token = 1*tchar
tchar = "!" / "#" / "$" / "%" / "&" / "'" / "*" / "+" / "-" / "." / "^" / "_" / "`" / "|" / "~" / DIGIT / ALPHA
ALPHA =  %x41-5A / %x61-7A   ; A-Z / a-z
DIGIT =  %x30-39 ; 0-9
```
So, it's only basic visible ASCII characters that are identical for all locales.